.storybook/main.js

```diff
@@ -1,11 +1,37 @@
+const path = require('path');
+
 module.exports = {
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
   addons: [
+    '@storybook/addon-docs',
     '@storybook/addon-links',
-    // '@storybook/addon-essentials',
+    '@storybook/addon-essentials',
     '@storybook/preset-create-react-app',
+    'storybook-addon-react-docgen',
```

Comment:
This is a pretty old plugin and isn't really necessary with the updated Controls API. Wondering if there's a specific reason for including this?

Reply:
Can you elaborate?  I have approved this, and merged, but I can always yank the line.  I believe it was left there because we didn't know what it was used for, and were adding it back in after we figured out the "real problem" preventing builds.  Earlier we had just nuked all doc elements.,